test/subset.jl

```diff
@@ -0,0 +1,143 @@
+module TestSubset
```

Comment:
Can you add tests with `GroupedDataFrame`?

Reply:
added, ported from `@where`.